colorcoder/ColorCoding.java

```diff
@@ -0,0 +1,28 @@
+package colorcoder;
+
+public class ColorCoding {

```

Comment:
The functionality encapsulated in the ColorCoding doesn't match with class name(ColorCoding)
when all the members in the class is static - should class is  final?